tests/xtro-sharpie/watchOS-UserNotifications.ignore

```diff
@@ -0,0 +1,7 @@
+# both added and deprecated in Xcode 13 beta 2
```

Comment:
@tj-devel709  do we get these errors when using beta 3? Have you tried removing it?

Reply:
When I remove this, I get errors still!